data/en/mbs_0203.json

```diff
@@ -0,0 +1,215 @@
+{
+    "title": "Monthly Business Survey",
+    "survey_id": "009",
+    "mime_type": "application/json/ons/eq",
+    "theme": "default",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
+    "view_submitted_response": {
+        "enabled": true,
+        "duration": 900
+    },
+    "sections": [{
+            "id": "section",
+            "groups": [{
+                "id": "msi",
+                "title": "",
+                "blocks": [{
+                        "type": "Introduction",
+                        "id": "introduction",
+                        "primary_content": [{
+                            "type": "Basic",
+                            "id": "use-of-information",
+                            "content": [{
+                                "list": [
+                                    "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.",
+                                    "You can provide informed estimates if actual figures aren’t available.",
+                                    "We will treat your data securely and confidentially."
+                                ]
+                            }]
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
```

Comment:
its not like this in slides and I think QBS is the same where there is only 1 question so doesnt use twistees and looks really weird as nothing in it, i would just add as a line of text so less weird

Reply:
I've put it into a single item list area that we already have in the pattern and removed it from the accordion.